std/random.d

```diff
@@ -1772,11 +1772,51 @@ else
     }
 }
 
+version (linux)
+{
+    // `getrandom()` was introduced in Linux 3.17.
+
+    // Shim for missing bindings in druntime
+    version (none)
+        import core.sys.linux.sys.random : getrandom;
+    else
+    {
+        import core.sys.posix.sys.types : ssize_t;
+        extern extern(C) ssize_t getrandom(
+            void* buf,
+            size_t buflen,
+            uint flags,
+        ) @system nothrow @nogc;
+    }
+}
+
 /**
 A "good" seed for initializing random number engines. Initializing
 with $(D_PARAM unpredictableSeed) makes engines generate different
 random number sequences every run.
 
+This function utilizes the system (CS-)PRNG where available and implemented
```

Comment:
Use "(cryptographically secure) pseudo-random number generator" and introduce the acronym, and then use it throughout (rather than using the acronym first, and then spelling it out later).

Reply:
Updated to:
> This function utilizes the system *cryptographically-secure pseudo-random
> number generator (CSPRNG)* or *pseudo-random number generator (PRNG)*
> where available and implemented (currently `arc4random` on applicable BSD
> systems or `getrandom` on Linux) to generate “high quality” pseudo-random
> numbers – if possible.